src/borg/archiver/diff_cmd.py

```diff
@@ -105,6 +108,9 @@ def build_parser_diff(self, subparsers, common_parser, mid_common_parser):
         )
         subparser.add_argument("--sort", dest="sort", action="store_true", help="Sort the output lines by file path.")
         subparser.add_argument("--json-lines", action="store_true", help="Format output as JSON Lines. ")
+        subparser.add_argument(
+            "--content-only", action="store_true", help="Only compare differences in content (Exclude time differences)"
```

Comment:
**exclude** (is in the middle of a sentence, no reason to uppercase that).
**timestamp* (more precise than *time*)

Reply:
That's more accurate.